### PR TITLE
Use radio buttons for selecting templates

### PR DESCRIPTION
**Pull request by bjorvack**
## Type

- Enhancement

## Pull request description

Renders the template choices using radiobuttons

### CODE REVIEW

File: src/Backend/Modules/Pages/Resources/views/FormLayout.html.twig [hunk 1]
```diff
@@ -293,34 +293,16 @@
...
-              </div>
-            </div>
-
-            {# loop 2 example #}
-            <div class="col-lg-6">
-              <div class="form-check mb-3">
-                <input class="form-check-input" type="radio" name="templateRadios" id="templateRadio2" value="templateName2" checked>
-                <label class="form-check-label" for="templateRadio2">
-                  Template name <br>
+            {% for template in form.templateId.children %}
```
Comment (carakas) [inline src/Backend/Modules/Pages/Resources/views/FormLayout.html.twig:296]: Why not show the visualisation of the template anymore?

**Review by carakas — CHANGES_REQUESTED** [review]

File: src/Backend/Modules/Pages/Resources/views/FormLayout.html.twig (hunk 1 above)
Comment (bjorvack) [reply src/Backend/Modules/Pages/Resources/views/FormLayout.html.twig:296]: We do, this is done on line 301

**Review by Katrienvh — APPROVED** [review]

File: src/Backend/Modules/Pages/Resources/views/FormLayout.html.twig [hunk 2]
```diff
@@ -293,34 +293,16 @@
...
+              <div class="col-lg-6">
+                <div class="form-check mb-3">
+                  {{ form_widget(template) }}
                   <div class="template-visual-small">
-                    {{ selectedTemplate.html|raw }}
+                    {{ template.vars.attr['data-html']|raw }}
```
Comment (carakas) [inline src/Backend/Modules/Pages/Resources/views/FormLayout.html.twig:301]: if you add in in the attr field, does it then not also appear on as an attribute on the checkbox?

**Review by carakas — CHANGES_REQUESTED** [review]

**Comment by carakas** [comment]
I think it is better to create a custom form type and pass it like this https://github.com/forkcms/forkcms/blob/master/src/Backend/Form/Type/MetaType.php#L307 and then use the id of the template as key or something

File: src/Backend/Modules/Pages/Resources/views/FormLayout.html.twig (hunk 2 above)
Comment (bjorvack) [reply src/Backend/Modules/Pages/Resources/views/FormLayout.html.twig]: @carakas changed in bd842e9ac91cadb60b24b0718a163ffe61bfbbed

**Comment by bjorvack** [comment]
@carakas It wasn't lazy loaded. I've updated the code in 96de0a2

**Pull request merged**